main.py

```diff
@@ -1,10 +1,44 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
```

Comment:
Do you need a blank line between these imports?

Reply:
I re-read the rules - no need